roles/sonarqube/templates/values/00-main.j2

```diff
@@ -23,7 +32,9 @@ ingress:
       pathType: Prefix
       path: "/"
   annotations:
+{% if not cluster_k8s %}
```

Comment:
```suggestion
{% if cluster_openshift %}
```
Globalement toutes les parties spécifiques à openshift (securityContext et autres) sont ajoutées à cause d'openshift, et pas retirés à cause de k8s donc je serai d'avis d'inverser tous les `if not k8s_cluster` en `if openshift_cluster`

Reply:
Mais le paramètre `cluster_k8s` est justement introduit par la PR du fait que l'installation dans un cluster OpenShift est le comportement par défaut.
Si nous introduisons un paramètre `cluster_openshift`, nous devrons en l'état le spécifier à chaque installation en tant qu'extra variable.
Je propose plutôt d'introduire un paramètre `clusterType` dans la section `global` de la dsc, et de le positionner par défaut sur la valeur "openshift" au niveau du fichier ".roles/socle-config/files/cr-conf-dso-default.yaml".
Il faudrait également introduire ce paramètre dans la section global de la CRD (fichier ".roles/socle-config/files/crd-conf-dso.yaml") en tant que paramètre requis, positionné par défaut sur "openshift" et proposant l'alternative "k8s".
Ainsi, la condition pour tous les paramètres spécifiques à OpenShift deviendrait `{% if dsc.global.clusterType == "openshift" %}`.
Qu'en pensez-vous ?

Reply:
C’est en effet plus logique selon comment on se positionne.
Je me positionnais comme si le déploiement du socle sur OpenShift était le standard, c’est pourquoi mon code était orienté pour que cela soit transparent à l’actuel, tout en ouvrant des possibilités.
Si le standard devient de déployer sur Kubernetes Vanilla, il faudrait alors dans la logique, tout inverser.

C’est en effet une bonne idée de définir dans "roles/socle-config/files/crd-conf-dso.yaml" plutôt que passer par des externals vars.